docs/integrations/code-conventions.md

```diff
@@ -600,6 +600,34 @@ You may also use ```headerTransform``` to convert the existing keys into formatt
  
  <img width="758" src="../doc_imgs/integrations/50575199-fd5d0a00-0e01-11e9-9d54-944eb7c6f287.png"></img>
 
+### IndicatorsTimeline
+The `IndicatorTimeline` is an optional object (available from Server version 5.5.0 and up) . It is only applicable for commands that operate on indicators. It is a dictionary (or list of dictionaries) of the following format:
+When `IndicatorTimeline` data is returned in an entry, the timeline section of the indicator whose value was noted in the timeline data will be updated (and is viewable in the indicator's view page in Cortex XSOAR as can be seen in the attached image).
+
+<img width="758" src="../doc_imgs/integrations/timeline_section.png"></img>
+
+**What value should be used for the `'Category'` field of a `timeline` data object?**  
+Any Cortex XSOAR integration command that returns `timeline` data should include the `'Category'` value of `'Integration Update'`. When returning `timeline` data from a Cortex XSOAR automation, the value passed to the `'Category'` field should be `'Automation Update'`.
+
+**So when should one include a timeline object in an entry returned to the war room?**  
```

Comment:
I think the better example is if we have command like `quarantine-endpoint` we should add a timeline that explains what happened to this host.
I am not sure if we want to timeline on reputation commands. If yes, then we can have it built in as part of the current api, update the timeline any time indicators provided with message like `Indicator ${x} was enriched`
But I leave the decision to @michalgold and @cloughrm . What do you think?

Reply:
We should capture the enrichment result on the timeline - this would be useful. This is especially true for network based indicators - imagine an analyst viewing the results for an IP or Domain when researching a security incident which may have happened weeks ago - enrichment information over time would be a big differentiator for us vs competition.

Reply:
I didn't know that we are planning to include the whole enrichment result in the timeline. Timeline has message. Message usually for simple messages like `8.8.8.8 was enriched by VirusTotal` `192.12.34.12 was quarantined` and not for big JSON results from virus total. 
If this is the case, let have meeting , because I want better to understand the use case

Reply:
@anara123 - by "enrichment result" im specifically referring to the reputation score for the integration. So timeline would look like:

<timestamp> | VirusTotal reputation "Bad"
<timestamp> | RecordedFuture reputation "Suspicious"